dimagi/utils/django/cached_object.py

```diff
@@ -259,28 +259,7 @@ def cache_put(self, image_stream, metadata):
         rcache.set(self.meta_key(OBJECT_ORIGINAL), simplejson.dumps(image_meta.to_json()))
 
     def is_cached(self):
-        try:
-            metas, streams = self.get_all_keys()
```

Comment:
this only checked for meta anyways and when the cache is put, it is always put as `OBJECT_ORIGINAL` so checking for that key is equivalent